reminders.txt

```diff
@@ -22,4 +22,5 @@ Hey human, remember to believe in yourself ❤️
 Hey human, remember to distribute your today's task evenly so that you don't stress on things and get them done on time 📝
 Hey human, time management is the key 🔑.
 Hey human, remember to tell your heart that ALL IS WELL 💚. 
-Hey human, pursue for excellence and success wil chase you ✨.
\ No newline at end of file
+Hey human, pursue for excellence and success wil chase you ✨.
+Hey human, Just a reminder " Excuses Don't Get Results. "
```

Comment:
I’d make it a bit more gentle like:

Hey human, just remember - no excuses! Dw, you got this 💪